light_controller/light_controller.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import sys
+import time
+
+from light_controller.ir_toy.ir_toy import IR_Toy
+
+__author__ = 'akm'
+
+
+class LightController:
+    """
+    IR Light Controller interface
+
+    Controls several RGB LED globes and strips that use a common 24 button
+    IR remote to change colours.
+
+    They use slightly different protocols you can record the buttons to store
+    in JSON which is loaded by this class.
+
+    :param comPort: Serial device to connect to.
+    """
+    buttonNames = (
```

Comment:
Is this just a combination of _colours and _commands? Maybe we can generate it programmatically so we don't have the same thing twice